src/test.js

```diff
@@ -500,6 +509,13 @@ function synchronize( callback, priority ) {
 	}
 }
 
+function randomNumberGenerator( seed ) {
```

Comment:
But reading jquery/api.qunitjs.com/pull/121, I see why special-casing numbers is important. It would slightly vary the above example:
```js
if ( typeof seed !== "number" || seed % 1 !== 0 ) {
	seed = parseInt( generateHash( seed ), 16 );
}
seed = (seed % 0x100000000) || -1;
```